README.md

```diff
@@ -1,9 +1,7 @@
 # ⛩ node-gateway
+[![Test](https://github.com/satsuma-xyz/node-gateway/actions/workflows/test.yml/badge.svg)](https://github.com/satsuma-xyz/node-gateway/actions/workflows/test.yml) [![GitHub release (latest by date)](https://img.shields.io/github/v/release/satsuma-xyz/node-gateway)](https://github.com/satsuma-xyz/node-gateway/releases) [![Docker Image Version (latest by date)](https://img.shields.io/docker/v/satsumaxyz/node-gateway?logo=docker)](https://hub.docker.com/r/satsumaxyz/node-gateway/tags) [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/satsuma-xyz/node-gateway/issues) [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/satsuma-xyz/node-gateway/blob/main/LICENSE)
```

Comment:
Could we have each badge on its own line for readability?

Reply:
IMO think it's too long to have one badge every line. How about batching them in a line by category? 

1. CI/CD
2. Releases
3. Contributions


Looks like this:
![image](https://user-images.githubusercontent.com/6441090/187238948-1c789f84-7425-4efe-8582-3aeeb8e2439f.png)

Reply:
Sorry, I meant in the code. Is there a way to have separate lines in code but same line in markdown?

Reply:
oh yeah you can!

Reply:
@brianluong Mind making this tweak? Looks pretty odd to have multi line badges.

Reply:
yeah sure!

Reply:
https://github.com/satsuma-xyz/node-gateway/pull/57